code/__DEFINES/_flags.dm

```diff
@@ -163,6 +163,10 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define LETPASSCLICKS (1<<13)
 /// Allows you to pass over windows and window-adjacent stuff, like windows and windoors. Does not include airlocks with glass in them.
 #define PASSWINDOW (1<<14)
+/// Allows you to pass over humans specifically, allowing you to still hit all other mobs.
+#define PASS_HUMAN (1<<15)
+/// Allows you to pass over minebots specifically.
+#define PASS_MINEBOT (1<<16)
```

Comment:
I can see `PASS_HUMAN` but `PASS_MINEBOT` is just a waste of a passflag slot (which we are approaching the limit of)

Reply:
It seems like it would be pretty easy to add some special code in `/obj/projectile/kinetic/prehit_pierce(atom/hitting)`, perhaps a typecache on the projectile which is "paths we ignore" or something

Reply:
that's actually what I was looking at rn lol